.travis.yml

```diff
@@ -0,0 +1,33 @@
+# the following values must be set in https://travis-ci.com/<github-repo>/settings
+# - DOCKER_REPO (eg. myrepo/myapp)
+# - DOCKER_USERNAME
+# - DOCKER_PASSWORD
+
+services: docker
+language: go
+
+branches:
+  only:
+    - master
+    - /^v\d+\.\d+\.\d+.*$/
+
+env:
+  - ARCH=amd64
+  - ARCH=arm32v7
+  - ARCH=arm64v8
```

Comment:
Does QEMU support for ARMv6 for RasPi 1 and Zero?

Reply:
Yes, I think it does. I can give it a try with arm32v6 and submit another pull request with a 4th arch option this week. I don't have a Pi 1 or Zero to test it on though.

Reply:
Yes, QEMU does support arm32v6. However I just remembered why I didn't add that option initially.

I was trying to stick to official node images, just for the sake of it, and the [arm32v6/node](https://hub.docker.com/r/arm32v6/node/) images only come in alpine, so you lose some of the systemd functionality that you're getting with the debian images like [arm32v7/node](https://hub.docker.com/r/arm32v7/node/).

It's something we can definitely work around, like adjust the project dependencies to work on alpine, or find another base image that includes systemd and node on arm32v6. Maybe something from the [Belana.io repo](https://www.balena.io/docs/reference/base-images/base-images/) since they might have what we need.

Personally I would suggest switching to alpine and removing the systemd dependency. Then we can switch v7 for v6 since it's forward-compatible with all raspberry pi variants and it keeps our base images small.

Keeping both v6 and v7 variants is possible, but then we would need to switch to long arch names in the docker image tags. We can't use `-arm` for both v6 and v7 so we'd start using `-arm32v6`, `-arm32v7`, `-arm64v8`, etc. for everything. Which in turn requires a slightly different process for the multi-arch manifest creation, but it's manageable. 

I'll let you think on it and decide. I'm happy to help with the required changes.

Reply:
You're totally right, we should not need systemd at all with Docker. It's meant to run with the CLI, not use socket activation. And it would be great to have the smaller Alpine base images.

Reply:
All done, we're looking at ~40MB per image now instead of ~360MB.

I also replaced arm32v7 with arm32v6 which should run on all variants of raspberry pi.

Reply:
Ah sorry I did this comment on the wrong line. 😔

Does this mean arm32v6 builds run (backwards compatible) on arm32v7 machines? Or is there some kind of magic switching between arm32v6 and arm32v7 builds when arm32v6 is specified?

Reply:
Your first thought is correct, arm32v6 will run on all raspberry pi variants. There are some minor optimizations in arm32v7 for pi2 and pi3 but it's not worth having them both supported separately.